admin/lib/sessions/add.js

```diff
@@ -0,0 +1,41 @@
+module.exports = add
+
+var _ = require('lodash')
+// var accountsFind = require('../accounts/find')
+var accountsFindAll = require('../accounts/find-all')
+var internals = {}
+internals.request = require('../../../utils/request')
+internals.deserialise = require('../../../utils/deserialise')
+internals.serialise = require('../../../utils/serialise')
+
+function add (state, options) {
+  if (!options || !options.username) {
+    return Promise.reject(new Error('options.username is required'))
+  }
+
+  // TODO: use accountsFind 而不是 accountsFindAll
+  //       after updating accountsFind to match admin README doc
+  // return accountsFind(state, {username: options.username})
```

Comment:
could you create an issue to fix `admin.accounts.find` and add the URL to the issue here? That’ll make it easier to fix later, and we won’t forget it. Also could you use the English "instead of", it will make it simpler to understand for other people that can’t read Chinese. I love the 而不是 though it looks pretty cool :)